saleor/dashboard/order/forms.py

```diff
@@ -252,18 +252,25 @@ class OrderNoteForm(forms.Form):
 
 
 class ManagePaymentForm(forms.Form):
```

Comment:
Wouldn't it be better to call it:
```suggestion
class BasePaymentForm(forms.Form):
```

So the user directly knows it's an abstract class?